src/libcmd/installables.cc

```diff
@@ -204,7 +204,7 @@ void SourceExprCommand::completeInstallable(std::string_view prefix)
             prefix_ = "";
         }
 
-        auto [v, pos] = findAlongAttrPath(*state, prefix_, *autoArgs, root);
+        auto [v, pos] = findAlongAttrPath(*state, prefix_, *autoArgs, root, false);
```

Comment:
This is for `InstallableAttrPath` which corresponds to the non-flake case. So we do want to call the functors here